201-jenkins-to-azure-container-registry/azuredeploy.json

```diff
@@ -246,17 +246,17 @@
     }
   ],
   "outputs": {
-    "Jenkins Dashboard": {
+    "jenkins_dashboard_url": {
```

Comment:
According to [best practices](https://github.com/Azure/azure-quickstart-templates/blob/4581d906df5a3918e05616b1be74591bfffd5489/1-CONTRIBUTION-GUIDE/best-practices.md), parameters should use camelCase.

They don't explicitly say _outputs_ should be camelCase, but that's what most templates do to be consistent.